tests/reference/asr-generics_list_01-39c4044.stderr

```diff
@@ -0,0 +1,5 @@
+semantic error: Type mismatch in binary operator; the types must be compatible
+  --> tests/../integration_tests/generics_list_01.py:33:12
+   |
+33 |     return x / k
```

Comment:
Update the test to cast things correctly.

Reply:
Updated.